GDJS/Runtime/pixi-renderers/pixi-filters-tools.js

```diff
@@ -22,13 +22,13 @@ gdjs.NightPixiFilter = function() {
       opacity: { type: '1f', value: 1 }
   };
 
-  PIXI.AbstractFilter.call(this,
+  PIXI.Filter.call(this,
```

Comment:
👍 Thanks for the upgrade, nice to see that the Pixi.js API has not changed a lot!

Reply:
The AbstractFilter actually still works, just displays the deprecation warning in the console :D Pixi API seems to be exceptionally stable (for the JS lib :P), but, of course, I do advice you to test it with more games before merging ;)